common/djangoapps/student/models.py

```diff
@@ -250,7 +250,7 @@ class Meta:  # pylint: disable=missing-docstring
     country = CountryField(blank=True, null=True)
     goals = models.TextField(blank=True, null=True)
     allow_certificate = models.BooleanField(default=1)
-    bio = models.TextField(blank=True, null=True)
+    bio = models.CharField(blank=True, max_length=3000, db_index=False)
```

Comment:
I'm a little confused by what is the right thing to do here. This Stack Overflow article says that CharFields can never be null so ``null=True`` is unnecessary:

http://stackoverflow.com/questions/8609192/differentiate-null-true-blank-true-in-django

About half the char fields in this model specify ``null=True`` and half don't. :-)